app/src/main/java/edu/mills/cs115/AZCategoryAdapter.java

```diff
@@ -8,28 +8,52 @@
 import android.view.ViewGroup;
 import android.widget.TextView;
 
-class CategoryAdapter extends RecyclerView.Adapter<CategoryAdapter.ViewHolder> {
-    private final static String TAG = "CategoryAdapter";
+/**
+ * AZCategoryAdapter implements the adapter for the RecyclerView in AZCategoryFragment.
```

Comment:
Do you know if she wants us to provide links to other Activity files? I noticed she does this in her Javadoc comments. Ex:
* AZCategoryAdapter implements the adapter for the RecyclerView in {@link AZCategoryFragment}